### PR TITLE
Manipulator

### LINKED ISSUE #32: Set up Raspberry-Robot Arm Communication Link
- [x] Send Object Coordinates to Robot Arm's Microcontroller
- [x] Act on the Coordinates using Inverse Kinematics to reach the target position
- [x] Establish two-way communication between Raspberry & Arm's microcontroller
- [x] Establish two-way communication between Raspberry & Mobile platform microcontroller

### PR DESCRIPTION
**Pull request by MwauratheAlex**
Fixes #32

### CODE REVIEW

File: resources/manipulator/manipuator/src/main.cpp [hunk 1]
```diff
@@ -180,26 +203,28 @@ void moveToPos(double x, double y, double z)
...
-  Serial.println("value of q1 is :");
-  Serial.print(q1);
-  Serial.println();
+  // Serial.println("value of q1 is :");
+  // Serial.print(q1);
+  // Serial.println();
   q1 = q1 + 13;
   // snail(q1, j2Servo); //////
  // moveServo(q1, j2Servo);
   delay(200);
```
Comment (codewithlennylen) [inline resources/manipulator/manipuator/src/main.cpp:224]: Why the delay?

File: resources/manipulator/manipuator/src/main.cpp [hunk 2]
```diff
@@ -108,12 +121,22 @@ void loop()
     // 4. Send coordinates to IK function
     // pickStatus = pick(x, y, z);
     moveToPos(x, y, z);
+    delay(1000);
```
Comment (codewithlennylen) [inline resources/manipulator/manipuator/src/main.cpp:124]: Why the delays??

**Review by codewithlennylen — APPROVED** [review]
Won't the delays make the arm slow?

File: resources/manipulator/manipuator/src/main.cpp (hunk 1 above)
Comment (MwauratheAlex) [reply resources/manipulator/manipuator/src/main.cpp:224]: 1. This delays are included to allow enough time for the gripper to open and close.
delay(1000);
 gServo.write(90);
 delay(1000);
 gServo.write(0);
 delay(1000);
 
 2.This serve no purpose in my view.
 Eric implemented the function so maybe you can ask him.
  delay(200);
  // if (z == 0)
  //moveServo(-(90 - a1), j2Servo);

  // Serial.println("value of q1 is :");
  // Serial.print(q1);
  // Serial.println();
  q1 = q1 + 13;
  // snail(q1, j2Servo); //////
 // moveServo(q1, j2Servo);
  delay(200);

**Pull request merged**